code/modules/projectiles/guns/energy/kinetic_accelerator.dm

```diff
@@ -259,6 +259,23 @@
 /obj/projectile/kinetic/mech
 	range = 5
 
+/obj/projectile/kinetic/mech/on_hit(atom/target, blocked, pierce_hit)
+	if(ishostile(target))
+		damage *= 1.5
+	return ..()
```

Comment:
this won't work, most hostile mobs are basic now. since lavaland equipment has a pressure check already, why not just up the damage var?